draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -11,9 +11,10 @@
 <?rfc subcompact="no" ?>
 
 <rfc category="std"
-     docName="draft-ietf-sidrops-aspa-verification-16"
+     docName="draft-ietf-sidrops-aspa-verification-18"
      submissionType="IETF"
      consensus="true"
+     updates="9234"
```

Comment:
I don't see where RFC9234 is updated

Reply:
@mitradir @cjeker 
I think you are right, especially, now that we are basically referring to BGP sessions in this draft in the context of ASPA rather than Roles. I have removed the "Updates RFC 9234" flag.